geometry_msgs/Pose pose
float32[3] alpha
float32[3] power_terms